app/admin/edit-user/edit-user.component.html

```diff
@@ -0,0 +1,56 @@
+<md-card class="default-card">
+	<span *ngIf="user.id === -1">{{ 'loading' | translate }}...</span>
+	<md-card-content *ngIf="user.id !== -1">
+		<h2>{{ 'edit user' | translate }} {{ user.name }}</h2>
+		<form>
+			<md-grid-list cols="2" rowHeight="4em">
+
+				<md-grid-tile>
+					<md-input [(ngModel)]="user.lastName" required type="text" id="lastName" placeholder="{{'last name' | translate}}" name="name"
+						#lastName="ngModel">
+						<md-hint [hidden]="lastName.valid" class="error-hint">
+							<span>{{ 'name required' | translate }}</span>
+						</md-hint>
+					</md-input>
+				</md-grid-tile>
+
+        <md-grid-tile rowspan="4">
+          <!-- temporary until upload-picture component can be used -->
+					<div style="height: 100%;">
+            <md-card class="Image" style="height: 80%;"></md-card>
+            <button md-raised-button color="primary" class="uploadButton" style="height: 20%;">{{'upload picture' | translate}}</button>
+          </div>
+				</md-grid-tile>
+
+				<md-grid-tile>
+					<md-input [(ngModel)]="user.firstName" required type="text" id="firstName" placeholder="{{'first name' | translate}}" name="firstName"
+						#firstName="ngModel">
+						<md-hint [hidden]="firstName.valid" class="error-hint">
+							<span>{{ 'first name required' | translate }}</span>
+						</md-hint>
+					</md-input>
+				</md-grid-tile>
+				
+				<md-grid-tile>
+					<md-input [(ngModel)]="user.email" disabled type="email" id="email" placeholder="{{'email' | translate}}" name="email" #email="ngModel">
+					</md-input>
+				</md-grid-tile>
+
+				<md-grid-tile>
+					<select class="btn btn-default dropdown-toggle" (change)="changeRole($event.target.value)">
```

Comment:
The selection for the role should have a label "Role" as the text fields already have.